src/mqt/bench/tket_helper.py

```diff
@@ -318,7 +333,8 @@ def get_mapped_level(
 
     native_gatenames = get_rebase(gate_set_name, True)
     native_gate_set_rebase = get_rebase(gate_set_name)
-    arch = Architecture(cmap)
+    cmap_converted = [(i, j) for (j, i) in cmap]
+    arch = Architecture(cmap_converted)
```

Comment:
The switching was by mistake because I just used the CoPilot suggestion. The parsing is necessary to match the expected type if `List[Tuple[int, int]]` while we have the `cmap` as `List[List[int]]` .

Reply:
Alright 👍🏼